From e39b588be2c4aafd9ae0d8001d87bd7f352c61a5 Mon Sep 17 00:00:00 2001
From: Hien Dang <hien.dang.eb@rvc.renesas.com>
Date: Tue, 31 May 2016 21:04:56 +0700
Subject: [PATCH 588/909] arm64: dts: r8a7796: Add the emergency node in
 thermal-zones

commit 42f59064af655158f7b4271f75fdd7cbbee05a8d from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

[dien.pham.ry: arm64: dts: r8a7796: Add CA53 for EMS target CPU]

Signed-off-by: Hien Dang <hien.dang.eb@rvc.renesas.com>
Signed-off-by: An Huynh <an.huynh.uj@rvc.renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 3f7dec3440d6..46564a18591b 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -3070,6 +3070,18 @@
 	};
 
 	thermal-zones {
+		emergency {
+			polling-delay = <1000>;
+			on-temperature = <110000>;
+			off-temperature = <95000>;
+			target_cpus = <&a57_1>,
+				      <&a53_0>,
+				      <&a53_1>,
+				      <&a53_2>,
+				      <&a53_3>;
+			status = "disabled";
+		};
+
 		sensor_thermal1: sensor-thermal1 {
 			polling-delay-passive = <250>;
 			polling-delay = <0>;
-- 
2.17.1

